README.md

```diff
@@ -7,14 +7,9 @@ Astropy Paper III
 Scope of the Paper
 ------------------
 
-<!-- The goal is to produce a brief and informative paper that covers major
-astropy principles not mentioned in the first paper, the core v2.0
-package, and infrastructure in the astropy project to support development
-in python.
-
-The draft of the paper can be found [here](https://github.com/astropy/astropy-v5.0-paper/blob/master-pdf/main.pdf). -->
-
-TBD
+Our goal is to produce a brief and informative description of major updates
+about the Astropy Project, drawing on relevant changes or news since 2018
+(v2.0).
```

Comment:
I am guessing you mean the paper version and not `astropy` version?
```suggestion
(paper v2.0).
```

Reply:
I think he is referring to the astropy version. This paper is supposed to correspond to v5.0.

Reply:
Yea, that's right. Maybe we should just remove the "(v2.0)" entirely?
```suggestion
```

Reply:
I pushed a separate fix to remove ambiguity!